### PR TITLE
fix: sass task was looping on windows, gen files ignored

### PR DESCRIPTION
**Pull request by mmacartney10**
@bmds fixed issue that is described in the title

### CODE REVIEW

File: task.js [hunk 1]
```diff
@@ -92,8 +92,12 @@ module.exports = function task(gulp, projectConfig, tasks) {
 	*	WATCH TASKS
 	* ---------------------*/
 
+	Object.keys(taskConfig.files).forEach(function setupTasksFromConfig(key) {
+		taskConfig.watch.push('!' + taskConfig.files[key].src);
+	});
+
 	gulp.task('watch:' + TASK_NAME, function watchTask() {
-		gulp.watch(
+		gulp.watch (
```
Comment (bmds) [inline task.js]: Why the extra space?

**Comment by bmds** [comment]
Appveyor has had a moment, i've cancelled all it's builds & am going through them slowly restarting them so as not to overwhelm it

**Comment by mmacartney10** [comment]
No I didn't end up adding one, probably needs one to check no generated file goes into the watch task. Could do with some help doing that if anyone can assist.

**Comment by mmacartney10** [comment]
@bmds can we merge this into master? its causing issues with Chester Zoo development

**Pull request merged**